File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int ans = INT_MIN;
        int i = 0;
        for(auto j:gain){
            ans = max(ans,i);
            i += j;
        }
        return max(ans,i);
    }
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        ans = float("-inf")
        i = 0
        for j in range(len(gain)):
            ans = max(ans,i)
            i += gain[j]
        
        return max(ans,i)
        